examples/hosts/app-integration/external-data/src/model/appModel.ts

```diff
@@ -13,7 +14,15 @@ import type { IAppModel, IAppModelEvents, ITaskList } from "../modelInterfaces";
  * responsibilities and functionality.
  */
 export class AppModel extends TypedEventEmitter<IAppModelEvents> implements IAppModel {
-    public constructor(public readonly taskList: ITaskList, container: IContainer) {
+    public constructor(
+        public readonly taskList: ITaskList,
+        container: IContainer,
+        private readonly runtime: IContainerRuntime ) {
         super();
+        this.runtime = runtime;
+    }
+
+    public readonly debugSendCustomSignal = (): void => {
```

Comment:
Nit: could this just be a regular method? Same with the interface declaration.

Reply:
As opposed to an arrow?  Arrow is nice (and I believe necessary here) when the method will be attached as an event handler to avoid unbound `this`.  The customer could also do the bind but this prevents mistakes.